src/main/java/creditsys/Customer.java

```diff
@@ -1,9 +1,14 @@
+import java.util.ArrayList;
+import java.util.List;
+
+import creditsys.User;
+
 public class Customer extends User {
     private String address;
     private String phoneNumber;
     private String sex;
     private int age;
-    // private vechicle[] 
+    private List<Vechicle> vehicles; 
```

Comment:
hey there is a typo on "Vechicle", it might be causing a compilation error, it is also on "this.vechicles" on the customer class